presto-main/src/test/java/io/prestosql/sql/planner/optimizations/TestWindowFilterPushDown.java

```diff
@@ -101,74 +158,81 @@ public void testFilterAboveWindow()
 
         // remove subplan if predicate on row number symbol can't be satisfied
         assertPlanWithSession(
-                "SELECT * FROM (SELECT name, row_number() OVER(ORDER BY name) FROM nation) t(name, row_number) WHERE row_number < 0",
+                String.format("SELECT * FROM (SELECT name, %s() OVER(ORDER BY name) FROM nation) t(name, ranking) WHERE ranking < 0", rankingFunction),
```

Comment:
static import format

Reply:
I forgot how much we like to static import everything in this project =).